### PR TITLE
fix: external dns role errors

### PR DESCRIPTION
**Pull request by tarrantro**
### Description

What is this PR about?

- fix external dns error while using remote role

### Context

why you want to apply this change/features/fix?

- fix external dns error while using remote role

### Comments

Any extra information we need to know about this PR?

---

@jina-ai/team-wolf

### CODE REVIEW

File: modules/general/external-dns/main.tf [hunk 1]
```diff
@@ -22,7 +22,7 @@ resource "helm_release" "external_dns" {
   name       = "external-dns"
   repository = "https://kubernetes-sigs.github.io/external-dns"
   chart      = "external-dns"
-  version    = "1.10.1"
```
Comment (winstonww) [inline modules/general/external-dns/main.tf:25]: There should not be a need for this version change?

**Review by winstonww — CHANGES_REQUESTED** [review]
Just a minor comment.

File: modules/general/external-dns/main.tf (hunk 1 above)
Comment (tarrantro) [reply modules/general/external-dns/main.tf:25]: not necessary

File: modules/general/external-dns/main.tf [hunk 2]
```diff
@@ -95,18 +93,24 @@ resource "helm_release" "external_dns" {
     type  = "string"
   }
 
+  set {
+    name  = "triggerLoopOnEvent"
```
Comment (winstonww) [inline modules/general/external-dns/main.tf:97]: no need as per [here](https://github.com/kubernetes-sigs/external-dns/issues/2636)

**Review by winstonww — DISMISSED** [review]

File: modules/general/external-dns/main.tf (hunk 2 above)
Comment (tarrantro) [reply modules/general/external-dns/main.tf:97]: that is needed, so the external dns is triggered by event, not loop. So as to reduce the API throttle. https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/aws.md#throttling

Comment (tarrantro) [reply modules/general/external-dns/main.tf:97]: we monitor service and ingress, not crd

**Review by subbuv26 — APPROVED** [review]
lgtm

**Pull request merged**